content/en/docs/tasks/security/authorization/authz-dry-run/index.md

```diff
@@ -0,0 +1,173 @@
+
+* Read the [Istio authorization concepts](/docs/concepts/security/#authorization).
+
+* Follow the [Istio installation guide](/docs/setup/install/istioctl/) to install Istio.
+
+* Deploy Zipkin for checking dry-run tracing results. Follow the [Zipkin task](/docs/tasks/observability/distributed-tracing/zipkin/)
+  to install Zipkin in the cluster. Make sure the sampling rate is set to 100 that allows you to quickly reproduce the trace span in the task.
+
+* Deploy Prometheus for checking dry-run metric results. Follow the [Prometheus task](/docs/tasks/observability/metrics/querying-metrics/)
+  to install the Prometheus in the cluster.
+
+* Deploy test workloads:
+
+    This task uses two workloads, `httpbin` and `sleep`, both deployed in namespace `foo`.
+    Both workloads run with an Envoy proxy sidecar. Create the `foo` namespace and deploy the workloads with the following command:
+
+    {{< text bash >}}
+    $ kubectl create ns foo
+    $ kubectl label ns foo istio-injection=enabled
+    $ kubectl apply -f @samples/httpbin/httpbin.yaml@ -n foo
+    $ kubectl apply -f @samples/sleep/sleep.yaml@ -n foo
+    {{< /text >}}
+
+* Enable proxy debug level log for checking dry-run logging results:
+
+    {{< text bash >}}
+    $ istioctl proxy-config log deploy/httpbin.foo --level "rbac:debug" | grep rbac
+    rbac: debug
+    {{< /text >}}
+
+* Verify that `sleep` can access `httpbin` with the following command:
+
+    {{< text bash >}}
+    $ kubectl exec "$(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name})" -c sleep -n foo -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
+    200
+    {{< /text >}}
+
+{{< warning >}}
+If you don’t see the expected output as you follow the task, retry after a few seconds.
+Caching and propagation overhead can cause some delay.
+{{< /warning >}}
+
+## Create dry-run policy
+
+1. Create an authorization policy with dry-run annotation `"istio.io/dry-run": "true"` with the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -n foo -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: deny-path-headers
+      annotations:
+        "istio.io/dry-run": "true"
+    spec:
+      selector:
+        matchLabels:
+          app: httpbin
+      action: DENY
+      rules:
+      - to:
+        - operation:
+            paths: ["/headers"]
+    EOF
+    {{< /text >}}
+
+1. Verify a request to path `/headers` is allowed because the policy is created in dry-run mode:
+
+    {{< text bash >}}
+    $ kubectl exec "$(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name})" -c sleep -n foo -- curl "http://httpbin.foo:8000/headers" -s
+    {{< /text >}}
+
+## Check dry-run result in proxy log
+
+1. The dry-run results could be found in the proxy debug log, it will be something like `shadow denied, matched policy ns[foo]-policy[deny-path-headers]-rule[0]`.
```

Comment:
could be found in the proxy debug log, it will be something like => can be found in the proxy debug log, similar to